package.json

```diff
@@ -5,9 +5,9 @@
   "directories": {
     "test": "tests"
   },
+  "private": true,
   "scripts": {
-    "env": "wp-env",
-    "postinstall": "cd settings && npm install --force && cd .. && wp-env start && wp-env run composer install",
```

Comment:
I'm wondering if this should remain, but just as the composer install? 

```
"postinstall": "wp-env run composer install",
```

Reply:
I was going to note this. Why is it needed?

Reply:
Nvm, appears related to the test framework.